demos/defender/defender_demo_json/report_builder.c

```diff
@@ -479,6 +521,45 @@ ReportBuilderStatus_t GenerateJsonReport( char * pBuffer,
         }
     }
 
+    /* Write custom metrics. */
+    if( status == ReportBuilderSuccess )
+    {
+        charactersWritten = snprintf( pCurrentWritePos,
+                                      remainingBufferLength,
+                                      JSON_REPORT_CUSTOM_METRICS_FORMAT_PART5,
+                                      pMetrics->pCustomMetrics->cpuUsageStats.upTime,
+                                      pMetrics->pCustomMetrics->cpuUsageStats.idleTime,
+                                      pMetrics->pCustomMetrics->memoryStats.totalMemory,
+                                      pMetrics->pCustomMetrics->memoryStats.availableMemory );
+
+        if( !SNPRINTF_SUCCESS( charactersWritten, remainingBufferLength ) )
+        {
+            LogError( ( "Failed to write custom metrics (part 5)." ) );
+            status = ReportBuilderBufferTooSmall;
+        }
+        else
+        {
+            remainingBufferLength -= charactersWritten;
+            pCurrentWritePos += charactersWritten;
+        }
+    }
+
+    if( status == ReportBuilderSuccess )
```

Comment:
Can this `}` be part of `JSON_REPORT_CUSTOM_METRICS_FORMAT_PART5`?

Reply:
Sure